internal/cli/inputs.go

```diff
@@ -17,13 +17,13 @@ import (
 const (
 	flagApp      = "app"
 	flagAppShort = "a"
-	flagAppUsage = "the remote Realm app name or id"
+	flagAppUsage = "Specify the name or ID of a Realm app"
 
 	flagProject      = "project"
-	flagProjectUsage = "the MongoDB cloud project id"
+	flagProjectUsage = "Specify the ID of a MongoDB Atlas project"
 
 	flagProduct      = "product"
-	flagProductUsage = "the Realm app product (can be specified multiple times)"
+	flagProductUsage = `Specify the Realm app product(s) (Allowed values: "standard", "atlas")`
```

Comment:
the quotes around flagProductUsage `str` are different than flagProjectUsage & flagAppUsage "str"
can we keep them consistent?